MyApplication/app/src/main/java/io/ckl/articles/modules/read/ReadActivity.java

```diff
@@ -34,25 +41,32 @@
     @BindView(R.id.readContent)
     TextView readContent;
 
+    @BindView(R.id.readCheckBox)
+    CheckBox readCheck;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_read);
         ButterKnife.bind(this);
 
-        setTitle(getIntent().getStringExtra("Label"));
-        readTitle.setText(getIntent().getStringExtra("Title"));
+        // Set the result as First User to indicate that the "Mark as Read" was not clicked
+        setResult(RESULT_FIRST_USER);
+
+        // Set all the Widgets info
+        setTitle(getIntent().getStringExtra(getResources().getString(R.string.extraLabel)));
```

Comment:
Please consider extract `getIntent()` into a variable to avoid multiple method calls